src/lib/globals.js

```diff
@@ -63,4 +63,16 @@ export const getDayString = (d) => {
   else if (d==="F"){return "Friday"}
   else if (d==="S"){return "Saturday"}
   else if (d==="Su"){return "Sunday"}
+}
+
+
+//Takes the char form show.day and returns a number M-Su ===1-7
+export const getDayInt = (d) => {
```

Comment:
I think this function already exists as an object above - `dayToInt`

Reply:
It's an object, but yeah we could use the object to perform the same function in the component, or i guess we could use the dayToInt object in the getDayInt() function in globals, idk either way